scripts/views/mapView.js

```diff
@@ -1,439 +1,391 @@
 (function closure(firebase) {
+
+  var eventHandler = {};
+  var townhallData;
   var map;
-  var google;
-  var infowindow;
+  
+  // Define an intial view for the US
+  var continentalView = function(w,h) { return geoViewport.viewport([-128.8, 23.6, -65.4, 50.2], [w, h]); };
+  var continental = continentalView(window.innerWidth/2, window.innerHeight/2);
 
-//draws map
-  window.initMap = function initMap() {
-    google = window.google;
+  function setMap(){
+    // Specify Mapbox default access token
+    var accessToken = 'pk.eyJ1IjoiZG1vcmlhcnR5IiwiYSI6ImNpejlid2Y1djAxNWsyeHFwNTVoM2ZibWEifQ.MlGaldJiUQ91EDGdjJxLsA';
 
-    // Initalize reusable infowindow
-    infowindow = new google.maps.InfoWindow({maxWidth: 200});
+    // Specify uploaded Mapbox Studio style URL
+    var styleURL = 'mapbox://styles/dmoriarty/cizljl81e000x2rqk280evaee';
```

Comment:
@DylanMoriarty If we just go with a single tone choropleth looking thing I doubt there will be a ton of confusion.  Something like https://bl.ocks.org/mbostock/4060606

Would be good to have a key as well.